content/docs/tasks/security/vault-ca/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Istio Vault CA integration
+description: Tutorial on how to plug in a Vault CA for issuing certificates in Istio.
+weight: 10
+keywords: [security,certificate]
+---
+
+This tutorial walks you through an example to plug in a Vault CA for issuing
+certificates in Istio.
+
+## Before you begin
+
+* Create a new Kubernetes cluster to run the example in this tutorial.
+
+* On GKE, you may set the value of the `CLUSTER`, `ZONE`, and `PROJECT`
+environmental variables based on your GKE cluster and run the following
+`gcloud` command to fetch the credentials for your cluster.
+
+    {{< text bash >}}
+    $ export CLUSTER=YOUR-CLUSTER-NAME
+    $ export ZONE=YOUR-CLUSTER-ZONE
+    $ export PROJECT=YOUR-GKE-PROJECT-NAME
+    $ gcloud container clusters get-credentials $CLUSTER --zone $ZONE --project $PROJECT
+    {{< /text >}}
+
+## Install Istio with SDS enabled
+
+1.  Use `git clone` to download the latest Istio source code.
```

Comment:
Again, can we assume customers have access to the full 1.1 release bits and that they need/want our source code?

Reply:
Revised to assume customers have access to the full 1.1 release bits. Removed downloading source code.